lib/websocket_server.rb

```diff
@@ -2,6 +2,8 @@ class WebsocketServer
   Pairing = Struct.new(:is_ws, :proxy)
 
   def initialize
+    @logger = Rails.env.production? ? $websocket_log : Logger.new(STDOUT)
```

Comment:
uh, you're right :disappointed: so is it okay if i use the `websocket.log` in both production and development?